### PR TITLE
Landing Page Layout

### PR DESCRIPTION
**Pull request by iLevyTate**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# Home Page
+# Achiev𝘼𝙄m
+
+![type:video](./assets/achievaimpromo.mp4)
+
+
+[![Tests Status](https://github.com/xyvora/achievaim/workflows/Testing/badge.svg?branch=main&event=push)](https://github.com/xyvora/achievaim/actions?query=workflow%3ATesting+branch%3Amain+event%3Apush)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/xyvora/achievaim/main.svg)](https://results.pre-commit.ci/latest/github/xyvora/achievaim/main)
```
Comment (sanders41) [inline docs/index.md:7]: We can remove these since this is a web page and not repo dev related.

```suggestion
```

**Review by sanders41 — COMMENTED** [review]
I'm getting a 404 on the favicon.com

**Comment by iLevyTate** [comment]
@sanders41 updated the .ico errors

**Review by sanders41 — APPROVED** [review]
LGTM!

**Pull request merged**